sup3r/preprocessing/batch_handling.py

```diff
@@ -1379,7 +1838,10 @@ def __next__(self):
                 output_features_ind=self.output_features_ind,
                 training_features=self.training_features,
                 smoothing=self.smoothing,
-                smoothing_ignore=self.smoothing_ignore)
+                smoothing_ignore=self.smoothing_ignore,
+                model_mom1=self.model_mom1,
```

Comment:
@bnb32 at some point we need to start breaking these files up we don't need to do this right now. Typical guidance is <1k lines per file.